index.ts

```diff
@@ -36,7 +44,7 @@ interface esbuildSvelteOptions {
      * A function to filter out warnings
      * Defaults to a constant function that returns `true`
      */
-    filterWarnings?: (warning: Warning) => boolean;
+    filterWarnings?: (warning: Warning) => warning is Warning;
```

Comment:
Why does this return value need to change?

Reply:
It doesn't my bad!